src/index.js

```diff
@@ -28,6 +29,7 @@ ReactDOM.render(
         <Route path={`${defaultPath}default`} component={Main} />
         <Route path={`${defaultPath}searchbox`} component={SearchBox} />
         <Route path={`${defaultPath}autocomplete`} component={Autocomplete} />
+        <Route path={`${defaultPath}heatmap`} component={Heatmap} />
```

Comment:
Same as above

Reply:
not sure if I understood what you meant by align, so I sorted the import lines by length in https://github.com/google-map-react/google-map-react-examples/pull/19/commits/d9ff68f7d25dfd4e3e9bb3ce991aa7edfc39f357